rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -361,6 +361,8 @@ extern void pre_main (void);
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE / sizeof(uint32_t)];
 #elif defined(TARGET_XDOT_L151CC)
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE * 6 / sizeof(uint32_t)];
+#elif defined(TARGET_REALTEK_RTL8195AM)
```

Comment:
Can you list tests that are making trouble? Does the GDMA have to use main stack memory? @0xc0170  what do you think about disabling the GDMA?